converters/fromZigbee.js

```diff
@@ -4854,6 +4854,32 @@ const converters = {
             };
         },
     },
+    E1E_G7F_action: {
+        cluster: 64528,
+        type: ['raw'],
+        convert: (model, msg, publish, options, meta) => {
+            // A list of commands the sixth digit in the raw data can map to
+            const lookup = {
+                1: 'on',
+                // Two outputs for long press. The eighth digit outputs 1 for initial press then 2 for each
+                // LED blink (approx 1 second, repeating until release)
+                2: 'up',
+                3: 'down', // Same as above
+                4: 'off',
+                5: 'on_double',
+                6: 'on_long',
+                7: 'down_double',
+                8: 'down_long',
```

Comment:
I didn't expect a `_long` here, what if the sixth digit is 8 and the 8 digit is 2, we will get a `down_long_long` here?

Reply:
Sorry I wrote 'down' instead of 'off' on those bottom lookup values. Fixed it now.

This remote has different behaviour for the on/off buttons as it does for the up/down buttons. I believe this was intentional as it enables the device to repeat the _long command until released, providing you better dimming control.

This is tested on HASS + NodeRed - 
On / Off buttons: Supports double and long press by sending their own value ( 5 - 8 in the lookup list)

Up / Down buttons: Doesn't support double press. Long press is supported but it sends a single press with the 8th number being 1, then after a second it will continue to send more single presses with the 8th number being 2 (regardless of how long you hold, it never goes to 3).

Due to this behaviour, unlike the Phillips Hue Dimmer Switch, the command/action is fired whilst the button is held down instead of on release. The user will need to use debounce: 1 in order for the initial press to be ignored.